packages/rum-core/src/boot/rumPublicApi.spec.ts

```diff
@@ -112,18 +113,34 @@ describe('rum public api', () => {
         deleteEventBridgeStub()
       })
 
-      it('init should accept empty application id and client token', () => {
+      it('should accept empty application id and client token', () => {
         const hybridInitConfiguration: HybridInitConfiguration = {}
         rumPublicApi.init(hybridInitConfiguration as RumInitConfiguration)
         expect(display.error).not.toHaveBeenCalled()
       })
 
-      it('init should force session sample rate to 100', () => {
+      it('should force session sample rate to 100', () => {
         const invalidConfiguration: HybridInitConfiguration = { sessionSampleRate: 50 }
         rumPublicApi.init(invalidConfiguration as RumInitConfiguration)
         expect(rumPublicApi.getInitConfiguration()?.sessionSampleRate).toEqual(100)
       })
 
+      it('should set the default privacy level received from the bridge if the not provided in the init configuration', () => {
+        const hybridInitConfiguration: HybridInitConfiguration = {}
+        rumPublicApi.init(hybridInitConfiguration as RumInitConfiguration)
+        expect((rumPublicApi.getInitConfiguration() as RumInitConfiguration)?.defaultPrivacyLevel).toEqual(
+          PRIVACY_LEVEL_FROM_EVENT_BRIDGE
+        )
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏What about passing the bridge privacy level to the `initEventBridgeStub`?
It would avoid to look into the stub implementation to understand where this value come from